prov/efa/src/rxr/rxr_msg.c

```diff
@@ -63,16 +63,22 @@ ssize_t rxr_msg_post_cuda_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_ent
 {
 	int err, tagged;
 	struct rxr_peer *peer;
+	int pkt_type;
+	ssize_t delivery_complete_requested;
```

Comment:
Can you change these from `ssize_t` to `bool`?

Reply:
Chaged to `bool`.